setup.py

```diff
@@ -1,35 +1,57 @@
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
 # Copyright (c) 2017 National Research Foundation (South African Radio Astronomy Observatory)
 # BSD license - see LICENSE for details
-from setuptools import setup, find_packages
+from future import standard_library
```

Comment:
@bngcebetsha and @amakhaba 
You want to remove this line as it is causing the build to fail 
https://travis-ci.org/ska-sa/nosekatreport/builds/580658530?utm_source=github_status&utm_medium=notification

As you are using `future` before the installation.

Reply:
Removed the import statement from `setup.py` but does not fix the problem.